.github/workflows/spmd_tests.yaml

```diff
@@ -39,17 +39,18 @@ jobs:
         run: |
           pytest --shard-id=${{ matrix.shard }} --num-shards=4 --cov=spmd test/spmd/
```

Comment:
You need to change here too, delete the `shard-id` and `num_shards` logic

Reply:
Thanks! Updated!